lib/rucio/rse/protocols/storm.py

```diff
@@ -55,7 +55,7 @@ def lfns2pfns(self, lfns):
 
         lfns = [lfns] if isinstance(lfns, dict) else lfns
         for lfn in lfns:
-            path = lfn['path'] if 'path' in lfn and lfn['path'] else self._get_path(scope=lfn['scope'].external,
+            path = lfn['path'] if 'path' in lfn and lfn['path'] else self._get_path(scope=lfn['scope'],
```

Comment:
isn't this needed for multi-vo?

Reply:
In the protocols we assume that it is always the external scope already. It's difficult here, since these methods are both used at the client and server, but since the main usage is on client we decided that we assume external scopes here.